CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 * Fixed a lint error in proxy classes when the 'minSdkVersion' of user's project is smaller than 11 (#3356).
 * Fixed a potential crash when there were lots of async queries waiting in the queue.
+* Fixed a bug that Realm transformer does not transform any field accesses in model's constructors (#3361).
```

Comment:
Fixed a bug causing the Realm Transformer to not transform field access in the model's constructors